packages/edit-post/src/components/manage-blocks-modal/category.js

```diff
@@ -23,6 +25,17 @@ function BlockManagerCategory( {
 	toggleVisible,
 	toggleAllVisible,
 } ) {
+	const settings = useContext( EditPostSettings );
+	const { allowedBlockTypes } = settings;
+	blockTypes = useMemo( () => {
+		if ( allowedBlockTypes === true ) {
+			return blockTypes;
+		}
+		return blockTypes.filter( ( { name } ) => {
+			return includes( allowedBlockTypes || [], name );
+		} );
+	} );
```

Comment:
I would think we need some dependencies (a second argument) here, at least `blockTypes` and probably `allowedBlockTypes` (or `settings`)?

Reply:
Updated 👍